src/Indicator.vala

```diff
@@ -264,6 +268,22 @@ public class Power.Indicator : Wingpanel.Indicator {
 
         return false;
     }
+
+    private bool battery_charged_notification () {
+      if (is_in_session) {
+          try {
+              var notification = new Notify.Notification (_("Battery Charged"), _("Consider unplug the charger"), "battery-full-charged-symbolic");
```

Comment:
Should be "Consider unplugging the charger"

Reply:
Instead of hard coding "Battery", I would also use the device name here and say `"%s fully charged"` to match the description in the indicator

Reply:
Waiting for cassidy feedback so I can change the text.

Reply:
Yeah I'm happy with:

```suggestion
              var notification = new Notify.Notification (_("Battery fully charged"), _("Unplugging the device now can extend battery life"), "battery-full-charged-symbolic");
```

Reply:
Done in 5f050a0